assets/css/main.scss

```diff
@@ -148,6 +148,23 @@ ul.contact-info li {
     }
 }
 
+table {
+    width: 100%;
+    max-width: 100%;
+    margin-bottom: 20px;
+
+    th, td {
+        padding-left: 8px;
+        padding-right: 8px;
+        padding-top: 5px;
+        padding-bottom: 5px;
+    }
+
+    tbody > tr:nth-of-type(2n+1) {
```

Comment:
I think all of this is duplicated with the `.table.table-striped` classes? And it's overridden by them (when they're applied) since they have a higher precedence selector.
![image](https://user-images.githubusercontent.com/458683/49365453-c69ab380-f6de-11e8-8a12-247069f1c90b.png)

Is there any reason to not just add `{:.table.table-striped}` to your tables? That would also avoid any potential concerns of affecting anywhere else tables might be used that don't want that styling (for whatever reason.)

Reply:
I will reverse it and use `{:.table.table-striped}`

Reply:
thanks, sorry for extra work :(